Encountify/Services/ScoreboardCreation.cs

```diff
@@ -37,7 +38,7 @@ public async Task<List<ScoreboardEntry>> CreateScoreboard(bool reversed = true)
                 results.Add(new ScoreboardEntry()
                 {
                     Name = group.Users,
-                    Score = group.Points.Sum(),
+                    Score = group.Points.Aggregate(0, (agg, next) => agg + next),
```

Comment:
I also used Aggregate in the same place in my latest PR so let me know if I should keep it or not:D

Reply:
This part seems like like a fancy way to write Sum(), your PR seems like a very fancy way to write foreach loop :D. So maybe this one XD